cla_public/libs/mailgun.py

```diff
@@ -0,0 +1,22 @@
+import requests
+from flask import current_app
+
+
+def send_ref_by_email(to, subject, email_body):
```

Comment:
I think you should rename this function - there's no mention of refs in the function.
Something like `send_email` would be better.